p2p/simulations/network.go

```diff
@@ -665,6 +678,12 @@ func (self *Network) Load(snap *Snapshot) error {
 		}
 	}
 	for _, conn := range snap.Conns {
+
+		if !self.GetNode(conn.One).Up || !self.GetNode(conn.Other).Up {
+			//in this case, at least one of the nodes of a connection is not up,
+			//so it would panic when connecting
```

Comment:
Why does the code panic? We should fix the panic and return an error instead, and then explicitly ignore that error below. Another option is to only include connections in a snapshot for which both nodes are still up.

Reply:
Actually this is indeed the behavior we get - it doesn't panic, but the frontend gets an error: "nodes not up: one 6d7331750df133e021465db235de38870231c6cc0be542df4b254e9eeb727055a55ea2705b2930862cde9e0b933494634861918e96677cd2628eab2bc8427f4a is not up↵"

But this means the upload of a snapshot fails because a preventable situation (up connection to a node down). BTW, I believe that if the upload of a snapshot contains a node down, that node will never be started again, thus it's kind of a zombie. I think.

Reply:
Ok, so can we ignore the specific error on the line below? For example:

```go
err := Connect(conn.One, conn.Other)
if err == ErrNodeDown {
	// ignore connection errors due to nodes being offline
	continue
} else if err != nil {
	return err
}
```

Reply:
I guess you mean here to introduce a new `ErrNodeDown` error type? Seems like much better, yes

Reply:
I was, but on reflection the current solution is also ok, let's just update the comment to not mention a panic (if there is a panic somewhere, fix it).

Reply:
Ok, comment updated